aten/src/ATen/templates/Functions.cpp

```diff
@@ -90,4 +92,72 @@ Tensor TensorMaker::make_tensor() {
    return IntArrayRef(zeros, 1);
  }
 
+Tensor add(const Tensor& self, const Tensor& other, const Scalar& alpha) {
+   return self.add(other, alpha);
+ }
+
+ namespace native {
```

Comment:
nit (you don't need to change anything): most of the codebase is mostly separated based on its namespace. Stuff in the `at::` namespace that's device-agnostic, like this file, lives in `aten/src/ATen`. Stuff that involves our native backends (cpu/cuda) under the `at::native::` namespace usually lives in `aten/src/ATen/native`. There's a `BinaryOps.cpp` file where we would probably realistically move all of the `at::native::` code if we actually wanted to land this